src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -263,114 +266,118 @@ export default function EnvironmentEditor({
             </div>
             <div className={classes.container}>
                 {scenes !== undefined && scenes.length !== 0 ? (
-                    <DragDropContext onDragEnd={onDragEnd}>
-                        <Droppable
-                            droppableId="droppable"
-                            direction="horizontal"
+                    <div
+                        style={{
+                            display: "flex",
+                            alignItems: "center",
```

Comment:
Adding this line allows the intro/concl card to be centred properly but it also causes the weird shifting when moving multiple scene cards. Not really sure why this happens but I'm sure it's something dumb